config/locales/devise.en.yml

```diff
@@ -1,84 +1,76 @@
-# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
-
+---
```

Comment:
So it looks like this tool completely regenerates the yml each time?

Its hard to tell exactly what is changing here before everything is being rearranged.  It seems like removing translation can be done without reformatting the whole file.

Could you perahaps ask it for a list of unused translations and remove them manually so we can see whats is changing?

If we want to do a bulk reformatting, I'd rather have that as a separate no-op change which doesn't actually do anything except formatting... and then maybe a precommit to go alongside so we can't have to repeat such changes.